include/h2o/http2_scheduler.h

```diff
@@ -122,6 +123,28 @@ int h2o_http2_scheduler_run(h2o_http2_scheduler_node_t *root, h2o_http2_schedule
  */
 int h2o_http2_scheduler_is_active(h2o_http2_scheduler_node_t *root);
 
+/**
+ * returns if the scheduler tree is linear (no node has multiple children) and
+ * the weight is monotonically decreasing (parent's weight is greater than or equal to
+ * child's)
+ *
+ * Note: Chromium's request satisfies this property
+ */
+int h2o_http2_scheduler_is_linear_monotonic(h2o_http2_scheduler_node_t *root);
```

Comment:
If we are to retain this function, I think it might better if the name included "chrome" (or "chromium").

The rationale is that what we are doing here is checking if the priority tree being build looks like it is Chrome, and the list being linear and monotonic is merely a necessary condition. We might want to also check that the weight values are the ones that would be sent by Chrome.

Reply:
My intention was that "monotonic" part would cover the property on weight (weight values would decrease monotonically as you walk through the nodes from the root), thought the word "weight" was omitted from the function name due to the length.

I have been reluctant to include "chrome" or "chromium" in symbol names because I thought it was too specific, but now I rethink that maybe it might be better to simply say "chromium" rather than pretend to be generic (like "linear" or "monotonic"), as it might be less confusing.

Reply:
This function went away in the latest revision.